_themes/single.md

```diff
@@ -10,9 +10,9 @@ images:
 info:
   Ported By: "[BesrourMS](https://github.com/BesrourMS)"
   Original By: "[FreeHTML5.co](http://freehtml5.co/preview/?item=single-free-html5-bootstrap-template)"
-  Last Updated: 4/07/16
+  Last Updated: 2916-04-07
 ---
 
 Single theme from FreeHTML5.co, ported to Pico.
 
-Single is a free html5 bootstrap template. It is a simple, beautiful and clean looking modern template that would fit perfectly for personal sites. This bootstrap theme is built with the latest technology such as HTML5, CSS3 and jQuery. We’ve included the sass file for you to easy to customize the template.
```

Comment:
Some of these descriptions were a little more "Copy and Paste" than the others. :unamused: :sweat_smile: